styles/components/column.css

```diff
@@ -0,0 +1,4 @@
+.column {
```

Comment:
Good job, however, if you don't use the classes `column__skills` and/or `column__interests`, then you can just delete the classes :)